src/agent/onefuzz-agent/src/tasks/report/crash_report.rs

```diff
@@ -33,11 +33,13 @@ pub struct CrashReport {
     pub call_stack_sha256: String,
 
     #[serde(default, skip_serializing_if = "Vec::is_empty")]
+    #[serde(deserialize_with = "deserialize_null_default")]
```

Comment:
Instead of a custom `deserialize_with`, these could be optional which matches the python side:

https://github.com/microsoft/onefuzz/blob/97cba8eda1caf25c0ed47bc11be13dcccc75451c/src/pytypes/onefuzztypes/models.py#L256-L259

Reply:
I implemented the changes to match the python side and other configs.
but I personally think that having `Option<Vec<string>>` is redundant since since `None` and `Some(vec![])` can be used to represent the absence of a list.